scripts/gen_vimdoc.py

```diff
@@ -141,7 +141,7 @@ def debug_this(cond, o):
         try:
             name = o.nodeName
             o = o.toprettyxml(indent='  ', newl='\n')
-        except:
+        except Exception:
             pass
```

Comment:
This is still a bit "ugly" IMHO (should only handle expected exceptions probably).

Reply:
Yeah, but `debug_this` is just a dev-time hack.